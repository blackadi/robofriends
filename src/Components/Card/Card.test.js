import { shallow } from 'enzyme';
import Card from "./Card";

it('Card JSX', () => {
    const wrapper = shallow(<Card />);
    expect(wrapper).toMatchSnapshot();
})