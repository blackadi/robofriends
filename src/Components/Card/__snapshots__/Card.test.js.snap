// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card JSX 1`] = `ShallowWrapper {}`;
